tests/spread/ci/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Build an LXD spread test image with TPM software
+manual: true
+artifacts:
```

Comment:
why are you declaring this artifact?

Reply:
Thanks for catching this! This was something I tested while I was playing around with the task.